### PR TITLE
Support for multiple drop-shadows per layer.

### PR DESCRIPTION
**Pull request by dirkschulze**
This adds support for multiple drop shadows per layer.

Generator JSON introduces a new property `dropShadowMult` which has an array of multiple drop shadows. However, it is not consistent with the use of `dropShadowMult` and `dropShadow` so I transform them all to a list in svgOMG.

### CODE REVIEW

File: svgOMGeneratorStyles.js [hunk 1]
```diff
@@ -222,9 +222,20 @@
...
+                ];
+                delete svgNode.style.fx.dropShadow;
+            }
+            if (svgNode.style.fx.dropShadowMulti) {
+                // PS exports filters in the opposite order, revert.
+                svgNode.style.fx.dropShadowMulti.reverse();
+                svgNode.style.fx.dropShadowMulti.forEach(function (ele) {
+                    color = ele.color;
+                    ele.color = omgUtils.toColor(color);
+                    ele.opacity = ele.opacity ? ele.opacity.value / 100 : 1;
```
Comment (jaredadobe) [inline svgOMGeneratorStyles.js]: This kind of thing should be a function some where

File: svgWriterFx.js [hunk 2]
```diff
@@ -163,42 +163,59 @@
...
-            var omIn = ctx.currentOMNode,
-                dropShadow;
-            if (omIn && omIn.style && omIn.style.fx) {
-                dropShadow = omIn.style.fx.dropShadow;
-            
-                if (dropShadow && dropShadow.enabled) {
-                    return true;
-                }
+            var omIn = ctx.currentOMNode;
+            if (omIn && omIn.style && omIn.style.fx && omIn.style.fx.dropShadowMulti) {
```
Comment (jaredadobe) [inline svgWriterFx.js]: This seems like you could extract it further to make it easier to drop in support for other languages. Pull this entire thing out to a 'isMultiLayerFxEnabled(omIn, 'dropshadowMulti')

Comment (dirkschulze) [inline svgWriterFx.js]: I actually planned to do some optimizations in this way to remove some functions.

File: svgWriterFx.js [hunk 3]
```diff
@@ -312,6 +328,13 @@
                 if (solidFill && solidFill.enabled) {
                     return true;
                 }
+
+                solidFill = omIn.style.fx.solidFillMulti;
```
Comment (jaredadobe) [inline svgWriterFx.js]: See above.

File: svgWriterFx.js (hunk 2 above)
Comment (jaredadobe) [inline svgWriterFx.js]: No time like the present :D

Comment (dirkschulze) [inline svgWriterFx.js]: All right!

**Comment by jaredadobe** [comment]
Also noticed that this commit includes .DS_Store files. You will want to update your .gitignore to ignore them and git rm the ones included in this commit.

**Comment by dirkschulze** [comment]
The last two patches are preparation. I will turn over the other filter effects as I edit them similarly to drop shadow.

**Comment by jaredadobe** [comment]
Looks good to me! Feel free to merge when ready.

**Comment by jaredadobe** [comment]
Oh wait. Do you want to drop a .DS_Store into the .gitignore to avoid you accidentally adding them again.

**Comment by jaredadobe** [comment]
Oh too late :D

**Comment by dirkschulze** [comment]
Heh, I wonder why we didn't create the project with this options already.

**Pull request merged**